content/doc/developer/tutorial-improve/update-parent-pom.adoc

```diff
@@ -19,6 +19,52 @@ The most recent parent POM files also provide additional static analysis like sp
 // Create the branch
 include::doc/developer/tutorial-improve/_create-a-branch.adoc[]
 
+
+== Maven configuration
+
+Before updating the parent POM (Project Object Model) using the "mvn -ntp version:update-parent" command, you must make sure that the Maven configuration files are correctly configured.
```

Comment:
```suggestion
Before you proceed with the parent POM update, you must make sure that Maven is installed on your machine and that it is configured correctly for Jenkins plugin development.
```
The abbreviation POM is explained above, no need to expand it again. No need to duplicate the `mvn` command either.